src/bench.c

```diff
@@ -11,6 +11,31 @@
 #include "util.h"
 #include "bench.h"
 
+void help(int default_iters) {
+    printf("Benchmarks the following algorithms:\n");
+    printf("    1. ECDSA signing/verification\n");
```

Comment:
These lines can be surrounded with `#ifdef ENABLE_MODULE_ECDH` etc to make it only print the relevant ones (same below).

Reply:
I think when @sipa said "same below", he meant that the same could be done in the "args:" list printed further down. This is still missing. Changes look good otherwise.

Reply:
Oh, I forgot about the args list. Thanks!